.eslintrc

```diff
@@ -44,11 +44,15 @@
     "static/"
   ],
   "settings": {
+    "import/core-modules": [ django ],
     "import/resolver": {
       "node": {
         "extensions": [".js", ".jsx"]
+      },
+      "webpack": {
```

Comment:
from this line on, this differs from .eslintrc in a4. because there we do not have webpack and obviously cannot ignore adhocracy4.